src/sql/workbench/common/actions.ts

```diff
@@ -253,7 +253,7 @@ export class BackupAction extends Task {
 		}
 		if (profile) {
 			const serverInfo = connectionManagementService.getServerInfo(profile.id);
-			if (serverInfo && serverInfo.isCloud) {
+			if (serverInfo && serverInfo.isCloud && profile.providerName.toLowerCase() === 'mssql') {
```

Comment:
@kburtram @alanrenmsft Has there been any discussion about an API to query a provider for actions it supports? This fix is ok in the short term but long term I imagine we'll want this to be something defined by the provider instead of relying of a bunch of hard-coded checks.

Reply:
I agree but today if the provider doent listen to certain message types we do get an error that message type is not supported by the provider

Reply:
@Charles-Gagnon yes, we've talked about feature discovery a few times.  This was the idea behind the capabilities service, but I think we only got as far as connection options.  It would be great to see that discussion revived.  Though certainly out of scope for this change, IMO.